indicator.lua

```diff
@@ -22,6 +22,9 @@ local function worker(args)
   local onclick = args.onclick
   local hidedisconnected = args.hidedisconnected
   local popup_position = args.popup_position or naughty.config.defaults.position
+  if args.skiproutes or args.skipcmdline then
+    args.skipvpncheck = true
```

Comment:
Probably skipvpncheck should be true by default? Mostly peoples don't use VPN, don't you think so? 

You mention that vpn check requires paswordless sudo for find, grep and wg. Would it break anything for default installation?

Reply:
Agreed, see new commit